app/controllers/users_controller.rb

```diff
@@ -0,0 +1,70 @@
+class UsersController < ApplicationController
+  before_action :logged_in_user, only: [:index, :edit, :update, :destroy]
+  before_action :correct_user,   only: [:show, :edit, :update]
```

Comment:
nope...`check_current_user` or something

Reply:
Naming came from the book, will modify it accordingly to be more expressive. ✓